Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -479,4 +479,170 @@ HRESULT _DWriteFontGetBoundingBoxesForGlyphs(
     });
 
     return ret;
-}
\ No newline at end of file
+}
+
+#pragma region CGFontCreateWithDataProvider helpers
+
+/**
+ * Custom implementation of IDWriteFontFileStream that implements Read in terms of an underlying CFDataRef
+ */
+class DWriteFontBinaryDataStream : public RuntimeClass<RuntimeClassFlags<WinRtClassicComMix>, IDWriteFontFileStream> {
+protected:
+    InspectableClass(L"Windows.Bridge.DirectWrite", TrustLevel::BaseTrust);
+
+public:
+    DWriteFontBinaryDataStream() {
+    }
+
+    HRESULT RuntimeClassInitialize(CGDataProviderRef dataProvider) {
```

Comment:
given our odd implementation of CGDataProvider, i'm curious about this.

---
In reply to: [88146706](https://github.com/Microsoft/WinObjC/pull/1386#discussion_r88146706) [](ancestors = 88146706)